guides/source/layout.html.erb

```diff
@@ -86,7 +86,7 @@
                   <div class="guides-section">
                     <dt><%= section['name'] %></dt>
                     <% finished_documents(section['documents']).each do |document| %>
-                    <dd><a href="<%= document['url'] %>"><%= document['name'] %></a></dd>
+                    <dd><a href="<%= document['url'] %>" <% if @path == document['url'] %> class="active" <% end %>><%= document['name'] %></a></dd>
```

Comment:
Could be a better way for writing this 
```suggestion
                    <dd><a href="<%= document['url'] %>" class="<%= 'active' if @path == document['url'] %>"><%= document['name'] %></a></dd>
```

Reply:
this will leave class="" for non active links

Reply:
thoughts on using [aria-current="page"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-current) instead of a class?

Reply:
what is the advantages over the class?

Reply:
It helps with accessibility, because it tells assistive technologies (like screen readers) which link represents the current page.

It's more semantically correct than a one-off `.current` or `.active` class. The implementation should stay mostly the same.

Reply:
thank you for the details. i agree with you
https://github.com/ahmedsayedabdelsalam/rails/pull/1
if you approve it i'll merge it with this branch